source/fuzz/transformation_add_parameter.cpp

```diff
@@ -24,37 +24,84 @@ TransformationAddParameter::TransformationAddParameter(
     : message_(message) {}
 
 TransformationAddParameter::TransformationAddParameter(
-    uint32_t function_id, uint32_t parameter_fresh_id, uint32_t initializer_id,
+    uint32_t function_id, uint32_t parameter_fresh_id,
+    std::map<uint32_t, uint32_t>&& call_parameter_id,
```

Comment:
As per our discussion the other day, consider using `std::map<uint32_t, uint32_t> call_parameter_id` if you want to use `std::move`. The effect is the same but API is better (recall `MakeDataDescriptor` and the fact that we are unable to pass anything other than rvalue).